### PR TITLE
Cashew cheese recipe.

### PR DESCRIPTION
**Pull request by mookerji**
/cc @zmagg

### CODE REVIEW

File: _posts/necessities/2018-06-22-cashew-cheese.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Mix together into a food processor and blend until smooth:
+- 500g soaked cashews (simmered and soaked overnight)
+- 1 tbsp miso
+- 2 tbsp fermented tofu
+- 1/4 cup pickle brine (lactofermented)
+- 2 tbsp sherry vinegar
+- 2 tbsp nutritional yeast (optional)
+
+Empty into a glass bowl, cover with a secured cheesecloth, and leave to ferment
+in a sunny area (probably for a few weeks?).
```
Comment (zmagg) [inline _posts/necessities/2018-06-22-cashew-cheese.md]: Mention that it gets dark on top?

File: _posts/necessities/2018-06-22-cashew-cheese.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+This is a recipe for a soft cashew cheese spread, in the style of what you may
+find from Miyoko's Creamery.
+
+Mix together into a food processor and blend until smooth:
+- 500g soaked cashews (simmered and soaked overnight)
+- 1 tbsp miso
+- 2 tbsp fermented tofu
+- 1/4 cup pickle brine (lactofermented)
```
Comment (zmagg) [inline _posts/necessities/2018-06-22-cashew-cheese.md:15]: Do we ever add some pickles themselves or nah?

File: _posts/necessities/2018-06-22-cashew-cheese.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: Cashew Cheese
+category: necessities
+---
+
+This is a recipe for a soft cashew cheese spread, in the style of what you may
```
Comment (zmagg) [inline _posts/necessities/2018-06-22-cashew-cheese.md:7]: You should mention how you can form it into a little nice package after it's done fermenting.

**Comment by zmagg** [comment]
Poke

File: _posts/necessities/2018-06-22-cashew-cheese.md (hunk 2 above)
Comment (mookerji) [reply _posts/necessities/2018-06-22-cashew-cheese.md:15]: Nope

File: _posts/necessities/2018-06-22-cashew-cheese.md (hunk 1 above)
Comment (mookerji) [reply _posts/necessities/2018-06-22-cashew-cheese.md]: Fixed

**Comment by mookerji** [comment]
@zmagg updated!

**Pull request merged**